templates/bind/named.conf.options.j2

```diff
@@ -83,3 +83,21 @@ acl our_neighbors {
 {%   endfor %}
 {% endif %}
 };
+
+{% if bind9_logging_enable %}
+logging {
+  channel bind_log {
+    file "{{ bind9_log_path }}/bind.log" versions {{ bind9_log_versions }} size {{ bind9_log_size }};
+    severity {{ bind9_log_severity }};
+    print-category yes;
+    print-severity yes;
+    print-time yes;
+  };
+  category default { bind_log; };
+  category update { bind_log; };
+  category update-security { bind_log; };
+  category security { bind_log; };
+  category queries { bind_log; };
```

Comment:
Maybe we can make this a variable as well? The default can be `bind_log`, if you like, but so you can set it like:
```
category default { {{ bind_log_category }} };
and 
bind_log_category: "default_syslog; bind_log" 
```
What do you think?

Reply:
I think it's a good idea use a variable for the log category.

What do you think about [this](https://github.com/UdelaRInterior/ansible-role-bind9/commit/1f6c141580ddca818b1882049fd2bc448a232906#diff-7eeda618087b49ae876084ab6c73fdbbR88) implementation using a list?